api_admin_panel/app/components/query.py

```diff
@@ -0,0 +1,193 @@
+
+def form_item_entry(data: dict[str, str]):
+
+    def create_entry(title: str, function: callable):
+        return (
+            rx.input(
+                placeholder=title,
+                width="100%",
+                on_change=function,
+                variant="surface",
+            ),
+        )
+
+    return rx.hstack(
+        create_entry("key", lambda key: QueryState.update_keyy(key, data)),
+        create_entry("value", lambda value: QueryState.update_value(value, data)),
+        rx.button(
+            "DEL",
+            on_click=QueryState.remove_entry(data),
+            color_scheme="ruby",
+            variant="surface",
+            cursor="pointer",
+        ),
+        width="100%",
+        spacing="1",
+    )
+
+
+def form_item(
+    title: str, state: list[dict[str, str]], func: callable, event_trigger: callable
+):
+    return rx.chakra.accordion(
+        rx.chakra.accordion_item(
+            rx.chakra.accordion_button(item_title(title)),
+            rx.chakra.accordion_panel(
+                item_add_event(event_trigger),
+                width="100%",
+                display="flex",
+                justify_content="end",
+            ),
+            rx.chakra.accordion_panel(
+                rx.vstack(rx.foreach(state, func), width="100%", spacing="1")
+            ),
+        ),
+        allow_toggle=True,
+        width="100%",
+    )
+
+
+def form_body_param_item(
+    state: list[dict[str, str]], func: callable, event_trigger: callable
+):
+    return rx.chakra.accordion(
+        rx.chakra.accordion_item(
+            rx.chakra.accordion_button(item_title("Body")),
+            rx.chakra.accordion_panel(
+                rx.match(
+                    QueryState.current_req,
+                    (
+                        "GET",
+                        rx.select(
+                            QueryState.get_params_body,
+                            default_value="None",
+                            width="100%",
+                        ),
+                    ),
+                    (
+                        "POST",
+                        rx.vstack(
+                            rx.hstack(
+                                item_add_event(event_trigger),
+                                width="100%",
+                                justify_content="end",
+                            ),
+                            rx.select(
+                                QueryState.post_params_body,
+                                default_value="JSON",
+                                width="100%",
+                            ),
+                            rx.vstack(
+                                rx.foreach(state, func), width="100%", spacing="1"
+                            ),
+                            width="100%",
+                        ),
+                    ),
+                ),
+            ),
+        ),
+        allow_toggle=True,
+        width="100%",
+    )
+
+
+def form_request_item():
```

Comment:
We can move this from the accordion and place it at the same level as the run button. Something like this:
![image](https://github.com/reflex-dev/templates/assets/36110765/bc73c334-9cc4-48f5-bbee-c2268e5d3127)